theano/d3viz/d3viz.py

```diff
@@ -0,0 +1,99 @@
+"""Dynamic visualization of Theano graphs.
+
+Author: Christof Angermueller <cangermueller@gmail.com>
+"""
+
+import os
+import os.path as pt
```

Comment:
Please don't rename like this it's just confusing and only saves two characters.